### PR TITLE
[WIP] Pointshop Update

### PR DESCRIPTION
**Pull request by PYROTEKNIK**
i made an attempt to improve the overall design
image

### CODE REVIEW

File: gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua [hunk 1]
```diff
@@ -153,48 +216,116 @@ end
...
+
+        for k, v in pairs(tab) do
+            newtab[string.lower(v["alpha-2"])] = v
+        end
+
+        CountryData = newtab
+    end)
 end
 
+GetCountryData()
```
Comment (swampservers) [inline gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua]: Can you not do this extraneous http request until someone has actually moused over the thing or whatever

Comment (PYROTEKNIK) [reply gamemodes/cinema/gamemode/modules/scoreboard/cl_playerlist.lua]: i don't think there's an event for when tooltips appear

File: lua/autorun/client/stylesheet.lua [hunk 2]
```diff
@@ -112,4 +288,159 @@ function draw.GradientShadowUp(x, y, w, h, alpha)
     if cornerMat:IsError() then return end
     predrawshadow(alpha)
     surface.DrawTexturedRectUV(x, y, w, h, 1, 0, 1, 1)
-end
\ No newline at end of file
+end
+
+surface.CreateFont('SS_Heading', {
```
Comment (swampservers) [inline lua/autorun/client/stylesheet.lua:352]: Can you put swampshop-specific styles in a different file? lua/swampshop/stylesheet.lua (and make sure to include it)

Comment (PYROTEKNIK) [reply lua/autorun/client/stylesheet.lua:352]: i kinda figured most of this would probably be useful to reuse elsewhere

**Comment by swampservers** [comment]
Thank you!

**Pull request merged**